Automattic/wp-calypso
# Add support for different user for Gutenberg edge testing

#### Changes proposed in this Pull Request

* This PR adds the option to use a different user for the Gutenberg tests when the `GUTENBERG_EDGE` env variable

#### Testing instructions
* Decrypt updated config file and ensure that the new `e2eflowtestinggutenbergsimpleedge` user is present
* Run Gutenberg tests and make sure they run with the user `e2eflowtestinggutenbergsimple`
* Set `GUTENBERG_EDGE=true` and run the Gutenberg tests again. Make sure they now use `e2eflowtestinggutenbergsimpleedge` 

Related to the work being done in https://github.com/Automattic/wp-e2e-tests-gh-bridge/pull/82
